CONTRIBUTING.md

```diff
@@ -99,6 +99,15 @@ You may type this line on your own when writing your commit messages. However, i
 
 Forgot to add DCO to a commit? Amend it with `git commit --amend -s`.
 
+### Troubleshooting Failed DCO Checks
+
+Note that the DCO workflow app requires signatures on every commit that's part of a PR.
+
+If you've already signed all your commits and your PR still fails the DCO check, it's likely because the email address you signed the commit with doesn't match the one on your GitHub account. There are a couple ways to solve this:
+
+1. **If you want your email address to be public:** 1. Go to https://github.com/settings/emails and uncheck "Keep my email address private". 2. Following the [GitHub documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git), set your commit email address to your primary GitHub email address.
+2. **If you want your email address to remain private:** 1. Go to https://github.com/settings/emails and make sure "Keep my email address private" is checked. 2. Following the [GitHub documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git), set your commit email address to the `*@users.noreply.github.com` email listed in the "Keep my email addresses private" setting.
```

Comment:
So DCO says that we're supposed to use real email addresses, I am not sure we can provide this guidance. Let me check.

Reply:
This section it endorses using email addresses that cannot be contacted.

Not being reachable (via email) seems to be the opposed to an open-source philosophy.  I do not think we should include this section.  @joshuarrrr what is your take on this?

Reply:
I agree that we should not endorse this, but we should also not expect them to have their github email id be the one linked to the PR (Or make it public in the first place). The actual DCO requirement we have is just that they use an email id. Its not required that the Email match the one in github. Thats why i suggest updating the DCO checker rather than suggesting this.

Reply:
@peternied You're correct that the GitHub aliased email addresses are not receivable (I confirmed GitHub will not forward messages sent to those addresses. So I guess our organization perspective is that we only accept contributions from GitHub users who use their real names and email addresses publicly. I can see the argument from the accountability and liability standpoint even though it comes at the cost of potential contributions and participation. I will update to remove that option.